product_docs/docs/edbcloud/beta/administering_cluster/organization_and_access/02_users.mdx

```diff
@@ -1,24 +1,91 @@
 ---
-title: "Users"
+title: "Managing Users and Access"
 ---
 
-## Viewing Users
-If you have a role that grants you  access to Admin screen, you can view all users from your organization that has been logged in at least once.
+!!!
+Access to the Admin screen is required for these actions
 
-To view users:
-- Navigate to Admin > Users.
 
-The list of users is sorted by the time of the latest login.
+# Roles
+  
+## Default Roles
+Access to EDB Cloud UI is based on permissions. Roles are sets of permissions. Roles are a convenient abstraction to manage permissions assigned to users.
 
-!!! Note
-The preview release contains only the minimum viable functionality for managing users and role based access control.
+* The predefined _owner_ role created during marketplace sign up has all permissions.
+* Each organization will have two additional default roles of _reader_ and _contributor_ available during Preview.
+* These roles can be edited by changing their name, description, or permissions
 
-EDB will add search functionality as well as many other features for managing users in subsequent releases.
-!!!
+## Role Permissions
+
+Permissions are generally represented in the format _action:object_ where _action_ represents an operation that can be performed and _object_ represents a categoroy of EDB Cloud functionality. 
+
+The available _actions_ are:
+* _create _
+* _read  _
+* _update _
+* _delete_
+
+The available _objects_ are:
+* _databases_
+* _backups_
+* _events_
+* _permissions_
+* _roles_
+* _users_
+* _billing_
+* _versions_
+
+## Adding Roles
```

Comment:
Can we move this section "Adding Roles" -- > just not totally comfortable with the GA statement yet.

Reply:
I changed it to "in the future"

Reply:
i've deleted as requested